build.gradle

```diff
@@ -1,16 +1,22 @@
 buildscript {
     ext {
         springBootVersion = '2.1.2.RELEASE'
+        kotlinVersion = '1.3.50'
     }
     repositories {
         mavenCentral()
     }
     dependencies {
-        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+        classpath "org.springframework.boot:spring-boot-gradle-plugin:$springBootVersion"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
+        classpath "org.jetbrains.kotlin:kotlin-allopen:$kotlinVersion"
```

Comment:
Hmmm, all-open bych možná ještě nedával, pokud to není potřeba, když se ti tam podařilo dát kotlin-spring.

Reply:
Beru zpět, asi to generuje i Spring initilizr s tím all-open. Tak za mě OK.

Reply:
ok :)